channel.js

```diff
@@ -1081,4 +1082,22 @@ TChannel.prototype.isUnhealthyError = function isUnhealthyError(err) {
     return errors.isUnhealthy(codeName);
 };
 
+// TODO - make this do smart things
+TChannel.prototype.startSpan = function startSpan(head) {
```

Comment:
you probably need to pass `req`, not just `head`, because you want to start the new span as a child of the existing span, which will be stored in the `req`